django_project/realtime/tasks/headless/inasafe_wrapper.py

```diff
@@ -0,0 +1,234 @@
+# coding=utf-8
+"""Task for InaSAFE Headless."""
+import logging
+
+from realtime.tasks.headless.celery_app import app
+
+__copyright__ = "Copyright 2018, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+LOGGER = logging.getLogger('InaSAFE Headless')
+
+@app.task(name='inasafe.headless.get_keywords', queue='inasafe-headless')
```

Comment:
Shouldn't it start from `headless.get_keywords` instead?

Reply:
Well, I just want to make sure it's `inasafe`, so I put `inasafe.headless.get_keywords`. Is it mandatory to remove `inasafe`?

Reply:
mandatory since your inasafe-headless doesn't start from `inasafe` package.

Reply:
To be precise, your task name should be the same with the task name in inasafe-headless.
Since you don't put task name property, then it will follow default convention, which is package import. In this case, your task name would be `headless.tasks.inasafe_wrapper.get_keywords`

Reply:
@lucernae ah sorry, I added it in the `inasafe-headless` repo, see https://github.com/inasafe/inasafe-headless/pull/16/files#diff-f36ef00976776f9d5600e763f4ac0b8cR43

I read somewhere it's better to add the name for this kind of application (worker in another machine).